### PR TITLE
[feat] Emit event when fund contract is deployed #136

### LINKED ISSUE #136: [feat] Emit event when fund contract is deployed
### This issue will be part of ODHack9.0, please apply via Onlydust app
- Emit an event when a fund contract is successfully deployed by the fund manager contract.
- The event should contain: the owner (caller address), the new fund contract address and the id.
- Please read [contributors guide](https://github.com/web3wagers/gostarkme/blob/dev/CONTRIBUTORS_GUIDE.md) before asking for an issue

### Acceptance Criteria
- Code builds successfully.
- Event struct has all the attributes needed.

### PR DESCRIPTION
**Pull request by Gianfranco99**
# Pull Request

- [x] Closes #136 
- [ ] Added tests (if necessary)
- [x] Run tests
- [x] Run formatting
- [] Commented the code

## Changes description

What I did was add the fundDeployed event with the owner as the key. This event is emitted once the contract has been deployed successfully.

## Current output

image

## Time spent breakdown

Give a breakdown on the time you used to solve the issue you are closing.

## Comments

Any other details you might want to share.

### CODE REVIEW

File: contracts/src/fundManager.cairo [hunk 1]
```diff
@@ -60,7 +80,17 @@ mod FundManager {
                 self.fund_class_hash.read(), 12345, call_data.span(), false
             )
                 .unwrap();
-            self.funds.write(self.current_id.read(), new_fund_address);
+
+            self.funds.write(self.current_id.read(), address_0);
```
Comment (EmmanuelAR) [inline contracts/src/fundManager.cairo:84]: Please merge your branch with the latest changes because that variable address_0, is not the latest

Comment (EmmanuelAR) [reply contracts/src/fundManager.cairo:84]: Also change the reference of the address_0 in the event

**Review by EmmanuelAR — CHANGES_REQUESTED** [review]

**Comment by EmmanuelAR** [comment]
Thks so much for your work @Gianfranco99 just some changes!

File: contracts/src/fundManager.cairo (hunk 1 above)
Comment (Gianfranco99) [reply contracts/src/fundManager.cairo]: Done @EmmanuelAR !

**Review by EmmanuelAR — APPROVED** [review]

**Pull request merged**